src/libraries/Common/src/Interop/Unix/System.Native/Interop.ReadDir.cs

```diff
@@ -57,6 +57,7 @@ internal ReadOnlySpan<char> GetName(Span<char> buffer)
         internal static extern IntPtr OpenDir(string path);
 
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetReadDirRBufferSize", SetLastError = false)]
```

Comment:
`assert` in `SystemNative_GetReadDirRBufferSize` impl should be a `static_assert()`.